### DISCUSSION TITLE
`/i 2772` not giving anything, TShock v.5.2.1.0

### DISCUSSION THREAD

**Discussion by nikita-III** [Pryaxis/TShock#3046] [General]
1. TShock version: TShock for Terraria v.5.2.1.0
2. What commands I executed and what it showed me in chat after not giving me anything is shown on screenshots
3. (**Note that I didn't install any additional plugins or mods**) Previously I had a problem setting up the server: I tried every method that I could find and think of. In the process I edited the `config.json` file, `serverconfig.txt` (from `Terraria` folder) and `tshock.sqlite` (using `DB Browser (SQLite)`): I gave `tshock.item.give` and `tshock.item.spawn` permissions for groups that didn't have those permissions. But eventually I just started `TShock.Installer.exe` again, and it informed me that I forgot to use `exit` to shutdown the server (I had been trying to make the server work for about 2 days, I may haven't remember all the things that I did and that might be essential). Then I entered the server and started giving myself different items (but I accidentally typed in incorrect names and `/i platform`, which corresponded to multiple items). (**Essentially I wanted this server to be installed on my PC just for the ability to give any item in any quantity, so I didn't care much about lots of things.**)
4. 
What is showed in chat:
![tshock 5210 issue 2](https://github.com/user-attachments/assets/c3b9f791-6e30-479b-81d3-2789ca15e6b3)
What was in the console:
![tshock 5210 issue 1](https://github.com/user-attachments/assets/8f08cae7-ae2b-4637-bfe8-8db0570506f3)

**Comment by sgkoishi** [comment]
You can check [Luminite Axes](https://terraria.wiki.gg/wiki/Luminite_Axes) and they are [unobtainable](https://terraria.wiki.gg/wiki/Unobtainable_features). Specifically, Luminite Axes
> Even with inventory editors, they seem to disappear, leaving an empty slot.

**Reply by nikita-III** [reply]
Thanks!

**Comment by THEXN** [comment]
If you want the server to be able to generate these 'unobtainable' items, you can try using this plugin: https://github.com/THEXN/TShockPlugin/blob/master/src/UnseenInventory/README.md.
Note that it only allows for server-side generation; to keep the item, you will also need to make changes on the client side.

**Reply by nikita-III** [reply]
Thanks!
